server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -48,7 +48,9 @@ export function create(
 		throttleIdPrefix: (req) => getParam(req.params, "tenantId"),
 		throttleIdSuffix: Constants.alfredRestThrottleIdSuffix,
 	};
-	const generalTenantThrottler = tenantThrottlers.get(Constants.generalRestCallThrottleIdPrefix);
+	const perTenantThrottler = throttlersMap
+		.get("perTenant")
```

Comment:
this should be .get(Constants.perTenantThrottler)

Reply:
Done. :)